src/js/shared/status-constants.js

```diff
@@ -24,6 +25,7 @@ module.exports = {
     INCOMPLETE: 'incomplete',
     COMPLETE_YES: 'complete-yes',
     COMPLETE_NO: 'complete-no',
+    COMPLETE_EXPIRED: 'complete-expired',
```

Comment:
I would have said it has not been completed and is now expired but one can look at it either way. Let's leave it as is.